app/views/browse/deployment-config.html

```diff
@@ -8,7 +8,7 @@ <h1 class="contains-actions">
           <div class="pull-right dropdown" ng-if="deploymentConfig" ng-hide="!('deploymentConfigs' | canIDoAny)">
```

Comment:
We should be updating the `canIDoAny` checks to use the same resource versions as the actions